templates/commands/render/render.go

```diff
@@ -551,6 +602,8 @@ type stepParams struct {
 	scratchDir  string
 	stdout      io.Writer
 	templateDir string
+	// Temporary directory to hold debug information when debug is enabled.
```

Comment:
Could we note in the comment that this field is empty when debugging is not enabled?